doc/man/8/rbd.rst

```diff
@@ -65,10 +65,9 @@ Parameters
 
    Specifies the size (in M/G/T) of the new rbd image.
 
-.. option:: --order bits
+.. option:: --object-size B/K/M
 
-   Specifies the object size expressed as a number of bits, such that
-   the object size is ``1 << order``. The default is 22 (4 MB).
+   Specifies the object size in B/K/M, The default is 4 MB.
```

Comment:
Might be nice to note that the smallest object-size is 4K and that the size will be rounded up the nearest power of two.